tests/data/desc-002.params.json

```diff
@@ -0,0 +1,6 @@
+[
+    ["namespace-uri(/*)", "http://docbook.org/ns/docbook"],
+    ["local-name(/*)", "book"],
+    ["/*/@xml:lang", ["en"]],
+    ["/d:book/d:variablelist/d:varlistentry/d:listitem/d:para/text()", []]
+]
```

Comment:
You should also test, if the `term` element contains something. However, that depends on if you take into account this `objtype` that I've mentioned in my last comment.